_pages/dev/features/product-multi-dimensional.md

```diff
@@ -0,0 +1,193 @@
+
+***
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+  {:toc}
+
+***
+
+## Enabling Multi-Dimensional Products
+
+If you are setting up a new Spartacus app, you can enable multi-dimensional products through the schematics by selecting the either or both of the following features:
+
+- `Product Multi-Dimensional - Selector`, which enables the category variant selectors on the product details page
+- `Product Multi-Dimensional - PLP price range`, which enables the List Item Details component on the product list page, and which adds the price range to the list item.
+
+If you have already set up your storefront app, you can enable multi-dimensional products by installing the `@spartacus/product-multi-dimensional` library with the following command:
+
+```zsh
+ng add @spartacus/product-multi-dimensional
+```
+
+For more information, see [Installing Additional Composable Storefront Libraries](https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/cfcf687ce2544bba9799aa6c8314ecd0/e38d45609de04412920a7fc9c13d41e3.html?locale=en-US#loioaa76b408d0324aea889aeeaed899144a).
+
+## Multi-Dimensional Selector
+
+The multi-dimensional selector is enabled by configuring a CMS component. The following is an example from `product-multi-dimensional-selector-component.module.ts`:
+
+```ts
+    provideDefaultConfig(<CmsConfig>{
+    cmsComponents: {
+        ProductMultiDimensionalSelectorComponent: {
+            component: ProductMultiDimensionalSelectorComponent,
+            guards: [ProductMultiDimensionalSelectorGuard],
+        },
+    },
+}),
+```
+
+Multi-dimensional products need to have the `ProductMultiDimensionalSelectorComponent` CMS component present in the `/page` response, and they need to be rendered on the page.
+
+### Adding a Multi-Dimensional Selector Using ImpEx
+
+To add a multi-dimensional selector using ImpEx, you create a `ProductMultiDimensionalSelector` CMS flex component, and then add it to a page slot, such as the `ProductSummarySlot`.
+
+**Note:** The `ProductSummarySlot` is the recommended location for adding your `ProductMultiDimensionalSelector`, but you can add the `ProductMultiDimensionalSelector` to any page slot on the product details page (PDP) that you wish.
+
+Variant elements for multi-dimensional products appear in locations such as the following:
+
+- product details pages
+- cart items (for example, see the `CartItemComponent`)
+- account pages, such as the order page, the order history, wish list, cancellations and returns, and so on.
+
+**Note:** The `$contentCV` variable that is used in the following ImpEx examples, and which stores information about the content catalog, is defined as follows:
+
+```text
+$contentCatalog=powertools-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+```
```

Comment:
I added this note and code snippet because it seems to be a "standard" bit of information we include, for example on pages such as [this](https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/af7716ec6b264f328f99bfa684577ff1.html?locale=en-US) or [this](https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/e2ca37dd44d04157bfc9be3b9bffa5ef.html?locale=en-US).

One thing I've noticed (only now, after these examples have been up for a few years) is this: `$contentCatalog=powertools-spaContentCatalog`. Is this fine? Should it say `electronics` instead of `powertools`, or it's fine the way it is?

Reply:
I've used this
```java
$productCatalog=electronicsProductCatalog
$contentCatalog=electronics-spaContentCatalog
$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Online])[default=$contentCatalog:Online]
$componentRef=ProductMultiDimensionalSelector
```

Reply:
👍 🙏 🙂